### PR TITLE
doc: update definition for namespaces

### PR DESCRIPTION
**Pull request by Shabirmean**
Update namespace definition to not use _Virtual Clusters_ as a means of explaining namespaces. 

Motif: KubeCon NA 21 Talk on 
***"Beyond Namespaces: Virtual Clusters are the Future of Multi-Tenancy - Lukas Gentele, Loft Labs"***

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ weight: 30
 
 <!-- overview -->
 
-Kubernetes supports multiple virtual clusters backed by the same physical cluster.
-These virtual clusters are called namespaces.
+Namespaces in Kubernetes provides a mechanism for scoping cluster resources by name. Names of resources need to be unique within a namespace, but not across namespaces. Namespace based scoping is applicable only for namespaced objects _(e.g. Deployments, Services, etc)_ and not for cluster-wide objects _(e.g. StorageClass, Nodes, PersistentVolumes, etc)_.
```
Comment (chrisnegus) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md:13]:
```suggestion
Namespaces in Kubernetes provides a mechanism for scoping cluster resources by name. Names of resources need to be unique within a namespace, but not across namespaces. Namespace-based scoping is applicable only for namespaced objects _(e.g. Deployments, Services, etc)_ and not for cluster-wide objects _(e.g. StorageClass, Nodes, PersistentVolumes, etc)_.
```

Comment (sftim) [reply content/en/docs/concepts/overview/working-with-objects/namespaces.md:13]: I'd reword “Namespaces in Kubernetes provides a mechanism for scoping cluster resources by name” to “In Kubernetes, _namespaces_ provides a mechanism for isolating groups of resources within a single cluster”.

**Comment by sftim** [comment]
> It looks like this is your first PR to kubernetes/website :tada:.

Congratulations!
